client/landing/gutenboarding/components/signup-form/index.tsx

```diff
@@ -194,7 +196,7 @@ const SignupForm = ( { onRequestClose }: Props ) => {
 							onChange={ setEmailVal }
 							placeholder={ __( 'Email address' ) }
 							required
-							autoFocus={ true } // eslint-disable-line jsx-a11y/no-autofocus
+							autoFocus={ isMobile ? false : true } // eslint-disable-line jsx-a11y/no-autofocus
```

Comment:
Nit 😬 
```suggestion
							autoFocus={ !isMobile } // eslint-disable-line jsx-a11y/no-autofocus
```

Reply:
Thanks @razvanpapadopol, I love a good nit like this! 😃